src/routes/v0/tag/global.rs

```diff
@@ -34,15 +43,27 @@ pub struct HotTagsQuery {
 )]
 pub async fn hot_tags_handler(Query(query): Query<HotTagsQuery>) -> Result<Json<HotTags>> {
     info!(
-        "GET {TAG_HOT_ROUTE} skip:{:?}, limit:{:?}, max_tagger: {:?}",
-        query.pagination.skip, query.pagination.limit, query.max_taggers
+        "GET {HOT_TAGS_ROUTE} skip:{:?}, limit:{:?}, taggers_limit: {:?}, tagged_type: {:?}, from: {:?}, to: {:?}",
+        query.pagination.skip, query.pagination.limit, query.taggers_limit, query.tagged_type, query.from, query.to
     );
 
     let skip = query.pagination.skip.unwrap_or(0);
     let limit = query.pagination.limit.unwrap_or(40);
```

Comment:
I think, in all the params that there is an option to fetch unlimited result, we should set a maximun number
```rust
    let limit = query.pagination.limit.unwrap_or(40).min(50);
```